packages/components/src/Modal/ModalHeader/ModalHeader.tsx

```diff
@@ -36,70 +35,44 @@ import {
   FontSizeProps,
   FontWeightProps,
 } from '@looker/design-tokens'
-import { Close } from '@styled-icons/material/Close'
-import { IconButton } from '../../Button'
 import { Heading } from '../../Text'
-import { DialogContext } from '../DialogContext'
 
-export interface DialogHeaderProps
-  extends SpaceProps,
-    CompatibleHTMLProps<HTMLElement>,
-    FontSizeProps,
-    FontWeightProps {
-  children: ReactNode
-  /**
-   * Don't include the "Close" option
-   * @default false
-   */
-  hideClose?: boolean
-  /**
-   * Replaces the built-in `IconButton` (generally used for close) with an arbitrary ReactNode
-   */
-  detail?: ReactNode
-}
+export type ModalHeaderProps = SpaceProps &
+  CompatibleHTMLProps<HTMLElement> &
+  FontSizeProps &
+  FontWeightProps & {
+    children: ReactNode
```

Comment:
Can this get narrowed to `ReactChild`?

```suggestion
    children: ReactChild
```

Reply:
This breaks `PopoverHeader`

Reply:
Ok - poked around a bit and figured out what was happening here - the `PopoverHeader` needs to explicitly specify `children: ReactChild` rather than _just_ using `FC<...>` because `FC` (FunctionalComponent) adds `children?: ReactNode` by default if it's not already specified.